flexget/config_schema.py

```diff
@@ -436,13 +436,13 @@ def validate_properties_w_defaults(validator, properties, instance, schema):
     yield from BaseValidator.VALIDATORS["properties"](validator, properties, instance, schema)
 
 
-def validate_anyOf(validator, anyOf, instance, schema):
-    errors = BaseValidator.VALIDATORS["anyOf"](validator, anyOf, instance, schema)
+def validate_any_of(validator, any_of, instance, schema):
+    errors = BaseValidator.VALIDATORS["anyOf"](validator, any_of, instance, schema)
```

Comment:
I think leaving the camelcase in this file is fine, since it's emulating the jsonschema keywords.

Reply:
Oops, I conflicted with myself on this comment. I've changed my mind and think snake_case is nicer than noqaing them.